app/controllers/mod_warning_controller.rb

```diff
@@ -65,8 +67,7 @@ def lift
     @warning.update(active: false, read: false)
     @user.community_user.update is_suspended: false, suspension_public_comment: nil, suspension_end: nil
 
-    AuditLog.moderator_audit(event_type: 'warning_lift', related: @warning, user: current_user,
-                             comment: "<<Warning #{@warning.attributes_print} >>")
+    audit_warning('warning_lift', @warning)
```

Comment:
Let's call this `suspension_lift`; warnings are one-time messages and can't be lifted.  (I realize it was already like this when you got here.)